Dockerfile.el8

```diff
@@ -48,13 +48,15 @@ RUN ( \
    dnf install -y rpm-build rpmlint python3-pip python3-virtualenv rubygem-ronn \
                   dnf-plugins-core 'dnf-command(config-manager)' \
                  'dnf-command(builddep)' sudo rsync rpmdevtools; \
-   dnf config-manager --set-enabled powertools;
+   dnf config-manager --set-enabled powertools; \
+   dnf clean all;
```

Comment:
This `clean all` is not needed, it slows down the second `dnf install ` below

Reply:
It could be! Depends on the network environment and the upstream server being used.

The better way may be to merge those installs, so that the process will be fast, and the image will be slow?